src/main/java/com/coreos/jetcd/watch/WatchCreateException.java

```diff
@@ -1,16 +0,0 @@
-package com.coreos.jetcd.watch;
```

Comment:
WatchCreateException is used in the current(not this pr) watch impl. It will break the ci if i just delete this exception.